frontend/src/app/shared/components/file-links/file-picker-modal/file-picker-modal.html

```diff
@@ -0,0 +1,57 @@
+<div
+  class="spot-modal spot-modal_wide"
+>
+  <div class="spot-modal--header">
+    <span class="spot-modal--header_title">{{text.header}}</span>
+    <button
+      class="button button_no-margin"
+      (click)="openStorageLocation()"
+    >
+      <span>{{text.buttons.openStorage()}}</span>
+      <span class="spot-icon spot-icon_external-link"></span>
+    </button>
+  </div>
+
+  <div class="spot-divider"></div>
+
+  <div class="spot-modal--body spot-container">
+    <ng-container *ngIf="(loading$ | async) === false; else loadingTemplate">
+      <ul
+        class="spot-list spot-list_compact op-files-tab--file-list"
+        data-qa-selector="op-files-picker-modal--file-list"
+      >
+        <li
+          *ngFor="let file of storageFiles$ | async | sortFiles; let i = index;"
+          class="spot-list--item op-files-tab--file-list-item"
+          data-qa-selector="op-files-tab--file-list-item"
+          op-storage-file-list-item
+          [storageFile]="file"
+          (changeSelection)="changeSelection(file)"
+        ></li>
+      </ul>
+    </ng-container>
+
+    <ng-template #loadingTemplate>
+      <op-loading-file-list></op-loading-file-list>
+    </ng-template>
+  </div>
+
+  <div class="spot-action-bar">
+    <div class="spot-action-bar--right">
+      <button
+        class="button button_no-margin spot-action-bar--action"
+        [textContent]="text.buttons.cancel"
+        (click)="closeMe()"
+      ></button>
+      <button
+        class="button -highlight button_no-margin spot-action-bar--action"
```

Comment:
We've improved the `spot-action-bar` styles removing the need for `button_no-margin` on both of these buttons

Reply:
If this is a form `submit` button, then pleas wrap the modal in a `form` and use `(ngSubmit)` instead of `(click)` here. Otherwise, this button should also be `type="button"`

Reply:
I think, this cannot be a form with a typical submit, or we would have to introduce hidden form fields maintaining the selected, but not displayed items.

this comes from the fact, that we can change the content of the modal (the checkboxes) dynamically and have to maintain a selection state in the component.

Reply:
fixed css classes and added `type="button"`